### PR TITLE
Upgrade TensorFlow and CUDA

### LINKED ISSUE #225: Upgrade Tensorflow for CUDA 10 compatibility

### PR DESCRIPTION
**Pull request by vs74**
This quick PR aims to upgrade upgrade `TensorFlow to  1.13.1` and `CUDA to 10.0`.

Fixes #225

### CODE REVIEW

**Comment by vs74** [comment]
This PR is ready to be reviewed.

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ All notable changes to the AxonDeepSeg project will be documented in this file.
 - Remove `matplotlib.pyplot` from source code and refactor it to full OO plotting
 - Standardize path management to `pathlib` library
 - Shifted AxonDeepSeg from TensorFlow to Keras framework.
+- Upgrade CUDA to 10.0 and tensorflow to 1.13.1.
```
Comment (jcohenadad) [inline CHANGELOG.md]: Upgraded

Comment (vs74) [reply CHANGELOG.md]: Fixed in commit ed3fb4c

**Comment by vs74** [comment]
@jcohenadad, can you re-review this PR ?

**Comment by vs74** [comment]
Though through this PR, TF and CUDA were upgraded to 1.13.1 and 10.0 respectively. While working with ADS in development mode it encounters with the flowing error ( conflict with dependency `version of numpy ` I guess). 
Screen Shot 2019-07-09 at 12 49 10 PM
I think for the time being until we resolve this issue we should revert to `tensorflow 1.3.0`. What do you suggest ?

**Comment by mathieuboudreau** [comment]
That's odd, because if there was an unsolvable conflict, then the Travis-CI tests would be failing I believe. 

Have you tried creating a new/clean virtual conda env and trying in that? Maybe there's some residual libraries in your venv that are conflicting.

**Comment by mathieuboudreau** [comment]
Actually I just tried that and replicated your error, I'm looking into it.

**Comment by mathieuboudreau** [comment]
@vs74 can you pull the branch `227_hotfix` and see if this resolved your issue? If so, please approve #232

**Pull request merged**